cleanrl/ppo.py

```diff
@@ -159,15 +166,24 @@ def get_action_and_value(self, x, action=None):
     # env setup
     envs = gym.vector.SyncVectorEnv(
         [make_env(args.env_id, args.seed + i, i, args.capture_video, run_name) for i in range(args.num_envs)]
-    )
+    )  # type:ignore[abstract]
```

Comment:
SyncVectorEnv inherits from VectorEnv which inherits from Env. For older gym version (I'm currently on 0.23.1), Env is an ABC with abstract method `render` that is not overriden by any of the vector envs. Since it's fixed in the newest gym release and not our issue, I'm ignoring here.